File: Project_DoAnNhanhSOL/app/src/main/java/com/example/project_doannhanhsol/DTO/Love.java
package com.example.project_doannhanhsol.DTO;

public class Love {
    private int Id_love,Id_Account,Id_product;
    private String Notes;

    public Love(int id_love, int id_Account, int id_product, String notes) {
        Id_love = id_love;
        Id_Account = id_Account;
        Id_product = id_product;
        Notes = notes;
    }

    public int getId_love() {
        return Id_love;
    }

    public int getId_Account() {
        return Id_Account;
    }

    public int getId_product() {
        return Id_product;
    }

    public String getNotes() {
        return Notes;
    }
}

File: WebApplication3/WebApplication3/Models/Love.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication3.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Love")]
    public partial class Love
    {
        [Key]
        public int Id_love { get; set; }

        public int? Id_Account { get; set; }

        public int? Id_product { get; set; }

        public string Notes { get; set; }
    }
}